R/map_autographr.R

```diff
@@ -68,6 +75,9 @@ NULL
 #' @describeIn autographing Graphs a network with sensible defaults
```

Comment:
for parameters edge_color and edge_size, are we calling it tie or edge variable?